### PR TITLE
feat: enriches issuer and verifier sites

### PR DESCRIPTION
**Pull request by lanterno**
This has the improved websites that I've showed you this morning.  It implements the first item in [this ticket](https://github.com/c4dt/eid-demo/issues/2) 
It doesn't include the logging feature.

### CODE REVIEW

File: components/notImplemented.vue [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+const props = defineProps(['homePageURL'])
+const homePageURL = props.homePageURL
+
+</script>
+
+<template>
+  <div class="w-full flex flex-col items-center justify-center">
+    <div class="flex flex-col items-center justify-center">
+      <p class="text-3xl md:text-4xl lg:text-5xl text-gray-800 mt-20">Oh! Curious eyes!</p>
+      <p class="md:text-lg lg:text-xl text-gray-600 mt-8">We under your curiosity but this page is not necessary for the demo, thus it was not implemented.</p>
```
Comment (PascalinDe) [inline components/notImplemented.vue:11]:
```suggestion
      <p class="md:text-lg lg:text-xl text-gray-600 mt-8">We understand your curiosity but this page is not necessary for the demo, thus it was not implemented.</p>
```

File: pages/issuer/index.vue [hunk 2]
```diff
@@ -0,0 +1,73 @@
...
+        </svg>
+      </div>
+      <div class="px-4 mx-auto max-w-7xl sm:px-6 lg:px-8 relative">
+        <div class="max-w-xl mx-auto text-center">
+          <div class="inline-flex px-4 py-1.5 mx-auto rounded-full">
+            <p class="text-4xl font-semibold tracking-widest text-g uppercase">EPFL HR system</p>  for graduate affairs
+
+          </div>
+          <p class="mt-4 text-base leading-relaxed text-gray-600 group-hover:text-white">
+            This platform provides different activities for EPFL HR department.
```
Comment (PascalinDe) [inline pages/issuer/index.vue:30]:
```suggestion
            This platform provides different activities for ACME HR department.
```

File: pages/verifier/index.vue [hunk 3]
```diff
@@ -0,0 +1,66 @@
...
+        </svg>
+      </div>
+      <div class="px-4 mx-auto max-w-7xl sm:px-6 lg:px-8 relative">
+        <div class="max-w-xl mx-auto text-center">
+          <div class="inline-flex px-4 py-1.5 mx-auto rounded-full">
+            <p class="text-4xl font-semibold tracking-widest text-g uppercase">Leo Inc. HR</p>
+
+          </div>
+          <p class="mt-4 text-base leading-relaxed text-gray-600 group-hover:text-white">
+            This platform provides different activities for EPFL HR department.
```
Comment (PascalinDe) [inline pages/verifier/index.vue:29]:
```suggestion
            This platform provides different activities for ACME HR department.
```

File: layouts/epfl.vue [hunk 4]
```diff
@@ -0,0 +1,115 @@
+<template>
+  <html class="h-full bg-gray-100">
+    <body class="h-full">
+      <div class="min-h-full">
+        <Disclosure as="nav" class="bg-gray-800" v-slot="{ open }">
+          <div class="mx-auto max-w-7xl px-4 sm:px-6 lg:px-8">
+            <div class="flex h-16 items-center justify-between">
+              <div class="flex items-center">
+                <div class="flex-shrink-0">
+                  <img class="w-24" src="https://c4dt.epfl.ch/wp-content/themes/epfl/assets/svg/epfl-logo.svg?color=indigo&shade=500" alt="Your Company" />
```
Comment (PascalinDe) [inline layouts/epfl.vue:10]: change this to other logo

File: pages/issuer/index.vue [hunk 5]
```diff
@@ -0,0 +1,73 @@
...
+              fill-rule="evenodd" />
+        </svg>
+      </div>
+      <div class="px-4 mx-auto max-w-7xl sm:px-6 lg:px-8 relative">
+        <div class="max-w-xl mx-auto text-center">
+          <div class="inline-flex px-4 py-1.5 mx-auto rounded-full">
+            <p class="text-4xl font-semibold tracking-widest text-g uppercase">EPFL HR system</p>  for graduate affairs
```
Comment (PascalinDe) [inline pages/issuer/index.vue:26]:
```suggestion
            <p class="text-4xl font-semibold tracking-widest text-g uppercase">ACME HR system</p>  for graduate affairs
```

File: pages/issuer/index.vue [hunk 6]
```diff
@@ -0,0 +1,73 @@
...
+  },
+  {
+    title: 'Students',
+    url: '/issuer/students',
+    description: 'Manages Student Information and maintains their contact information.'
+  },
+  {
+    title: 'Certificate Issuance',
+    url: '/issuer/issue',
+    description: 'Manage the issuance of certificates upon request by students.\n' +
```
Comment (ineiti) [inline pages/issuer/index.vue:69]:
```suggestion
    description: 'Issue certificates upon request by students.\n' +
```

**Review by ineiti — CHANGES_REQUESTED** [review]
There are still some references to `epfl` in the code, for completeness you can change them to `acme`.

I would also put the svg-files into a separate directory instead of inlining them. This helps when reading the code: if I see `acme-logo.svg` or `acme-logo.png`, I know what image is there. If I see the svg-path, I have no idea...

**Comment by lanterno** [comment]
@PascalinDe @ineiti I  removed all mentions of EPFL (except the schema name. we can create a new schema later with a different name) 

I also applied the other minor fixed (typos and SVGs moved to separate files always)

**Review by PascalinDe — APPROVED** [review]

**Pull request merged**